lib/taskana-core/src/main/java/pro/taskana/classification/internal/models/ClassificationImpl.java

```diff
@@ -54,20 +55,20 @@ public void setIsValidInDomain(Boolean isValidInDomain) {
 
   @Override
   public Instant getCreated() {
-    return created;
+    return created != null ? created.truncatedTo(ChronoUnit.MICROS) : null;
```

Comment:
All Instants in other entities are truncated to MILLIS, I think this should be changed in this class too